build.gradle

```diff
@@ -362,7 +362,7 @@ jacoco {
     toolVersion = '0.8.9'
 }
 
-sourceSets.main.java.srcDir "${buildDir}/generated-src/swagger"
+sourceSets.main.java.srcDir "${buildDir}/generated-src/swagger/src/main/java"
```

Comment:
This also fixes an issue where VSCode could not import generated methods. This is the correct directory, IntelliJ is more permissive.